### DISCUSSION TITLE
Is there any way to model a Vector using MySQL, or is that limited to PostgreSQL with postgres-array?

### DISCUSSION THREAD

**Discussion by benze** [SeaQL/sea-orm#1413] [Q&A]
I'm new to SeaORM and trying to understand if/how it would be possible to model an ordered Vector in a DB schema and in an Entity for SeaORM.

For instance, given a data model that looks like:

```
struct Song{
  name: String,
  key: String,
  type: String
}

and

struct PlaySet{
   event: String,
   songs: Vec<Song>,
}
```

Is there a way to model Song in the db schema?  How do I manage the song order in the PlaySet::songs list?  
Looking at the examples/basic example doesn't have a similar data structure where order is critical.

**Comment by tyt2y3** [comment]
I think you can use JSON. Ref: https://github.com/SeaQL/sea-orm/blob/master/tests/common/features/json_vec.rs
